relayer/relays/beacon/header/syncer/syncer.go

```diff
@@ -295,10 +310,13 @@ func (s *Syncer) GetBlockRoots(slot uint64) (scale.BlockRootProof, error) {
 	if err != nil {
 		return blockRootProof, fmt.Errorf("fetch beacon state: %w", err)
 	}
-	isDeneb := s.protocol.DenebForked(slot)
+
+	forkVersion := s.protocol.ForkVersion(slot)
 
 	blockRootsContainer = &state.BlockRootsContainerMainnet{}
-	if isDeneb {
+	if forkVersion == protocol.Electra {
+		beaconState = &state.BeaconStateElectra{}
+	} else if forkVersion == protocol.Deneb {
 		beaconState = &state.BeaconStateDenebMainnet{}
 	} else {
 		beaconState = &state.BeaconStateCapellaMainnet{}
```

Comment:
The Capella branch can be deleted?

Reply:
https://github.com/Snowfork/snowbridge/pull/1283/commits/ac5203550bd1df76cfc569c88c77cad6cbae593b.